gm4_smelteries/data/gm4_smelteries/functions/check_cook_time.mcfunction

```diff
@@ -0,0 +1,5 @@
+# @s - smeltery if structure correct
+# called by check_ore
+
+execute store result score @s gm4_bf_data run data get block ^ ^ ^1 CookTime
```

Comment:
So I've been trying to figure out why this number is gotten at all and I can't seem to figure it out. What's it for?

Reply:
CookTime is the time that an item's been smelting in a furnace. This is used to detect exactly when (technically a tick before) the item finishes smelting to spawn the item in the hopper.

Reply:
Oh I'm blind. The score is literally used in the next line because we need to check a range of values. I did a search and didn't find the score in any other file and got confused.